From 17420c9c2caaca882eada38a4694907dacd96d4b Mon Sep 17 00:00:00 2001
From: Wilson Ding <dingwei@marvell.com>
Date: Tue, 7 Jun 2016 20:44:20 +0800
Subject: [PATCH 522/538] fix: spi: a3700: disable the fifo mode

Since SPI FIFO mode does not work, this WA is issued
to disable SPI FIFO mode temporarily.

Change-Id: Ic01edb80508759ad9fedec17064e0ee0758c0985
Signed-off-by: Wilson Ding <dingwei@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/30688
Tested-by: Star_Automation <star@marvell.com>
---
 arch/arm64/boot/dts/marvell/armada-37xx.dtsi | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-37xx.dtsi b/arch/arm64/boot/dts/marvell/armada-37xx.dtsi
index 61a932f..7db34e3 100644
--- a/arch/arm64/boot/dts/marvell/armada-37xx.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-37xx.dtsi
@@ -111,7 +111,6 @@
 				cell-index = <0>;
 				clock-frequency = <160000>;
 				max-frequency = <40000>;
-				fifo-mode;
 				interrupts = <0 0 4>;
 				status = "disabled";
 			};
-- 
1.9.1

